noodles/noodles/View + ViewModel/Channels/ChannelCollectionViewDataSource.swift

```diff
@@ -35,6 +35,7 @@ class ChannelCollectionViewDataSource: NSObject, UICollectionViewDataSource {
 
         if fakeViewModel[indexPath.row].1 == 0 {
```

Comment:
@eloisafalcao nesta linha você está utilizando um mock do ViewModel que está correto, porém está correndo risco de causar mais trabalho para você ou quem for trabalhar na integração com o ViewModel em si. Sugiro que use Structs que o Edgar criou nas telas seguintes. Inicialize um struct de, por exemplo, ChannelModel com atributos mockados para fazer seu código. Não há necessidade de fazer modificações agora nos ViewControllers que você já trabalhou mas nos próximos seria bom você usar os structs que o Edgar fez.